weave-js/src/components/ToggleButtonGroup/types.tsx

```diff
@@ -0,0 +1,6 @@
+export enum ToggleButtonGroupSizes {
+  small = 'small',
+  medium = 'medium',
+  large = 'large',
+}
+export type ToggleButtonGroupSize = keyof typeof ToggleButtonGroupSizes;
```

Comment:
do you need this type? can you just do `size: ToggleButtonGroupSizes` in `ToggleButtonGroup.tsx`